deploy/olm-catalog/ibm-auditlogging-operator/3.6.0/ibm-auditlogging-operator.v3.6.0.clusterserviceversion.yaml

```diff
@@ -49,7 +50,7 @@ metadata:
     capabilities: Basic Install
     categories: Security
     certified: "false"
-    containerImage: quay.io/opencloudio/ibm-auditlogging-operator:latest
+    containerImage: quay.io/opencloudio/ibm-auditlogging-operator:3.6.0
```

Comment:
should you be using the sha here instead?

Reply:
this is in the annotations for the ocp console ui to display. was following swati's pr